core/commonMain/src/Instant.kt

```diff
@@ -314,6 +356,17 @@ public expect fun Instant.plus(value: Int, unit: DateTimeUnit, timeZone: TimeZon
 public fun Instant.plus(value: Int, unit: DateTimeUnit.TimeBased): Instant =
     plus(value.toLong(), unit)
 
+/**
+ * Returns an instant that is the result of subtracting the [value] number of the specified [unit] from this instant.
+ *
+ * If the [value] is positive, the returned instant is earlier than this instant.
+ * If the [value] is negative, the returned instant is later than this instant.
+ *
+ * The return value is clamped to the platform-specific boundaries for [Instant] if the result exceeds them.
+ */
+public fun Instant.minus(value: Int, unit: DateTimeUnit.TimeBased): Instant =
+    plus(-value.toLong(), unit)
```

Comment:
Better `-(value.toLong())`

Reply:
Though it's likely the same, but the order of operations would be more explicit.

Reply:
Done.